lua/telescope/builtin/internal.lua

```diff
@@ -45,6 +45,73 @@ internal.builtin = function(opts)
   }):find()
 end
 
+do
+  internal.menu = function(opts)
+    if opts.callback == nil then
+      vim.cmd [[echoerr 'There is no callback']]
+      return
+    end
```

Comment:
Idea what if each leaf can have a callback attached to itself. Because i believe we don't want to do the same for every single leaf, or do we.

Reply:
You also might wanna consider insert a `..` value on each iteration. So we are able to go up one table. Obviously not on the top level :)

Reply:
Definitely a good idea, I was thinking of the exact same thing about the going back. We could have a global callback and then a callback for each leaf to override.